test_runner/src/main/kotlin/ftl/gc/GcAndroidTestMatrix.kt

```diff
@@ -28,6 +29,15 @@ object GcAndroidTestMatrix {
         value = this@toEnvironmentVariable.value
     }
 
+    @VisibleForTesting
+    internal fun TestSetup.setEnvironmentVariables(args: AndroidArgs, testConfig: AndroidTestConfig) = this.apply {
+        environmentVariables = if (args.environmentVariables.isNotEmpty() && testConfig is AndroidTestConfig.Instrumentation) {
```

Comment:
`args.environmentVariables.isNotEmpty()` is redundant check, because is nothing wrong with running `args.environmentVariables.map { it.toEnvironmentVariable() }` on empty args